global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "sdk": {
-    "version": "6.0.100",
```

Comment:
We should work out some ordering here. I would like to separate "using the .NET 8 SDK" from "generating NRTs" - just updating to use .NET 8 may cause some significant difficulties internally, for reasons I'd rather not go into.

I'd personally be okay with this repo not having any NRT-based C# code in - or possibly explicitly keep it present but not compiled for now. (I think it's good that we've generated it and can check that it *does* compile, but we might not want to check it in at the moment.)

Reply:
@JasonLunn Still waiting for feedback if I should mark it as experimental and how to proceed with the tests.

Reply:
I'm aligned with checking in as little generated code as possible, and I think separating the SDK update from the NRT feature is a great suggestion. I don't feel strongly about the naming question, and would defer to @haberman and / or @jskeet on that point.